ui/app/ducks/metamask/metamask.js

```diff
@@ -97,9 +90,9 @@ function reduceMetamask (state, action) {
         },
       }
 
-    case actions.COMPLETED_TX:
-      const stringId = String(action.id)
```

Comment:
Huh - this was _completely broken_ beforehand. it literally never worked, as `action.id` was always undefined.

Reply:
I see - this action had no effect upon the `metamask` store, but it still worked in the `app` store.

I'd suggest deleting this block instead of repairing it, as this could have unintended consequences. Deleting it seems fine - it's dead code.

Reply:
I like deleting code